evml/torch/regression_losses.py

```diff
@@ -1,20 +1,12 @@
+""" Torch losses for regression models """
+
 import numpy as np
 import torch
-from torch import nn
 import torch.nn.functional as F
 
 
```

Comment:
Can you add some references here, similar to how you did it for the keras code pointing to Amini?

https://github.com/hxu296/torch-evidental-deep-learning/blob/main/evidential_deep_learning/losses/continuous.py

https://github.com/dougbrion/pytorch-classification-uncertainty/blob/master/losses.py

And the updated version to Amini's original -- https://github.com/deargen/MT-ENet/blob/468822188f52e517b1ee8e386eea607b2b7d8829/mtevi/mtevi.py